src/shared/parameter_v2/examples/generated_dynamic_parameters.hpp

```diff
@@ -0,0 +1,351 @@
+#pragma once
+#include <iostream>
+#include <boost/program_options.hpp>
+#include "software/parameter/config.h"
+#include "software/parameter/numeric_parameter.h"
+#include "software/parameter/enumerated_parameter.h"
+
+class foo;
+class example;
```

Comment:
should we append Config to the end? `fooConfig` vs `foo`

Reply:
Done